daemon/src/projection.rs

```diff
@@ -1255,8 +1255,8 @@ pub struct CfdOrder {
     #[serde(with = "round_to_two_dp")]
     pub lot_size: Usd,
 
-    #[serde(rename = "leverage")]
-    pub taker_leverage_choices: Leverage,
+    #[serde(rename = "leverage_choices")]
+    pub leverage_choices: Vec<Leverage>,
```

Comment:
🔧 😛 

```suggestion
    pub leverage_choices: Vec<Leverage>,
```

Reply:
done in [`fc5ddaa` (#1965)](https://github.com/itchysats/itchysats/pull/1965/commits/fc5ddaae1ffb681dc356c07e7af26984900f8ca3)